lib/account/account-manager.js

```diff
@@ -83,6 +145,147 @@ module.exports = function (accountService, config, hotp, otDate, promise, random
 
 
     /**
+     * Creates a password hash using the same method
+     * an account holder used to create a password to login.
+     *
+     * @param {string} password
+     * @param {string} salt
+     * @return {Promise.<string>}
+     */
+    function hashPasswordAsync(accountFile, salt) {
+        return secureHash.hashAsync(accountFile.password + salt, accountConfig.passwordHash.secondary, true).then((hashedPassword) => {
```

Comment:
Weird, I took it out, but I must have left it in the tests, so it was still working. This has been corrected.